doc/interface/binding/generalized_ion_exchange.rst

```diff
@@ -106,38 +106,70 @@ Generalized Ion Exchange
 
 **Unit:** :math:`m_{MP}^{3} mol^{-1}`
 
-===================  =========================  
+===================  ========================= 
 **Type:** double     **Length:** NCOMP
 ===================  ========================= 
 
 ``GIEX_NU``
    Base value for characteristic charges of the protein; The number of
    sites :math:`\nu` that the protein interacts with on the resin
-   surface
+   surface. If more than NCOMP values are provided, :math:`\nu` is given
+   by a piecewise cubic polynomial. Data is expected in component-major
+   ordering.
 
-===================  =========================  
-**Type:** double     **Length:** NCOMP
-===================  ========================= 
+===================  ===============================  
+**Type:** double     **Length:** multiples of NCOMP
```

Comment:
Is this really NCOMP or rather NBOUND?

Reply:
It's `NCOMP`. Each component has at most 1 bound state. Entries of non-binding components are ignored (but must be present just in the other adsorption models).

Reply:
Ok, good! Otherwise setting up the config would be a nightmare! ^^